x/globalfee/ante/fee.go

```diff
@@ -202,7 +194,17 @@ func (mfd FeeDecorator) ContainsOnlyBypassMinFeeMsgs(msgs []sdk.Msg) bool {
 	return true
 }
 
-// GetMinGasPrice returns the validator's minimum gas prices
+func (mfd FeeDecorator) GetBypassMsgTypes(ctx sdk.Context) (res []string) {
+	mfd.GlobalMinFee.Get(ctx, types.ParamStoreKeyBypassMinFeeMsgTypes, &res)
+	return
```

Comment:
no need to write "return" here ?

Reply:
No. Go functions that return values require the `return` keyword.